docker/docker-compose.dev.yml

```diff
@@ -57,10 +57,6 @@ services:
             - SLACK_CLIENT_ID=${SLACK_CLIENT_ID}
             - SLACK_CLIENT_SECRET=${SLACK_CLIENT_SECRET}
             - SLACK_STATE_SECRET=${SLACK_STATE_SECRET}
-            - LANGCHAIN_TRACING_V2=${LANGCHAIN_TRACING_V2}
-            - LANGCHAIN_ENDPOINT=${LANGCHAIN_ENDPOINT}
-            - LANGCHAIN_API_KEY=${LANGCHAIN_API_KEY}
-            - LANGCHAIN_PROJECT=${LANGCHAIN_PROJECT}
```

Comment:
note to remember updating the docs

Reply:
docs: https://app.graphite.dev/github/pr/lightdash/mintlify-docs/60/Remove-LangChain-environment-variables-from-AI-Analyst-section

cloud: https://app.graphite.dev/github/pr/lightdash/lightdash-cloud/1131/chore-remove-LangChain-configuration-from-customer-deployments